packages/core/src/app/Presenter.ts

```diff
@@ -123,10 +123,11 @@ export class Presenter {
   /**
    * Abort the ongoing presentation process.
    */
-  public abort() {
+  public async abort() {
     if (this.state.current === PresenterState.Initial) return;
     this.abortController?.abort();
     this.state.current = PresenterState.Aborting;
+    await this.playback.reset();
```

Comment:
I believe this is not necessary. Calling `present` already resets the playback here:
https://github.com/motion-canvas/motion-canvas/blob/5bd7c8e4f047408b4efa64f7e62db4257f409c57/packages/core/src/app/Presenter.ts#L176

I tested it, and resetting the `finished` flag that you added to the `PlaybackManager` seems to solve the issue just fine but let me know if I missed something.

Reply:
Oh yes, I missed that `reset` was already called. Removed it